integration.schema.json

```diff
@@ -264,7 +264,7 @@
         "$type": "Core.Entities.Configs.Steps.MailConfiguration, Core",
         "Id": "",
         "From": "",
-        "To": "",
+        "Tos": [],
```

Comment:
`Tos` really didn't land when I looked at this, I know understand it's meant as plural. Why not keeping it To and just requiring an array. Will be much clearer imo.

Reply:
Changed